### PR TITLE
Video resizing

### PR DESCRIPTION
**Pull request by jarrett-long**
- Full screen mode is available
- Video resizes on window.resize 
- Shortened timeout rate.  It seems to recognize faces more quickly when the timeout is shorter.

### CODE REVIEW

File: web/app.js [hunk 1]
```diff
@@ -52,7 +76,7 @@
 
                 trackerEnabled = true;
                 log('tracker watching...');
-            }, 3000);
+            }, 300);
```
Comment (brbarnett) [inline web/app.js]: Can we keep this 3000 for now? I don't want to overload the API

**Review by brbarnett — CHANGES_REQUESTED** [review]
The video loads on my screen but it does not track my face

File: web/app.js (hunk 1 above)
Comment (jarrett-long) [reply web/app.js]: Yeah sure thing.  I've added a commit to change it back to 3000.

**Comment by jarrett-long** [comment]
I've committed a tweak that stops and re-initializes the tracker task after window resizing.  Before, the boxing would not properly surround a face after adjusting the screen size.  

I've also noticed that it's much better at recognizing a face when the tracker.setStepSize() value is lowered.

**Comment by jorupp** [comment]
If there are values that are useful to tinker with for testing, it maybe helpful to have inputs on the page somewhere to specify them.

File: web/app.css [hunk 2]
```diff
@@ -1,21 +1,76 @@
...
+  #video-overlay{
+    z-index: 2147483647;
+  }
+
   #video-snapshot {
-    margin-left: 600px;
-    margin-top: 120px;
+    right: 0;
+    bottom: 0;
+    margin: auto 1% 1% auto;
```
Comment (brbarnett) [inline web/app.css:20]: Haven't seen this before (1%). What's it do?

**Review by brbarnett — APPROVED** [review]
Thanks for your help

**Pull request merged**